index.html

```diff
@@ -1684,29 +1686,8 @@ <h2>The <dfn>record type</dfn> string</h2>
         <dt>An <dfn>external type name</dfn></dt>
         <dd>
           A {{DOMString}} representing a custom type for the
-          <a>external type record</a>.
-          The type must follow the [=external type name=] ABNF.
-          <pre class="abnf">
-            ext-type             = reg-name ":" custom-type
-            custom-type          = 1*(ALPHA / DIGIT / other)
-
-            DIGIT                = %x30-39
-            ALPHA                = %x41-5A / %x61-7A   ; A-Z / a-z
-            other                = "(" / ")" / "+" / "," / "-" / ":" / "=" /
-                                   "@" / ";" / "$" / "_" / "!" / "*" / "'" / "."
-          </pre>
-          The `reg-name` value is a [=host/registrable domain=] owned by the issuing
-          organization, a "`:`" and a type, e.g. "`w3.org:member`".
-          And additional ABNF exists for [=well-known type records=]:
-          <pre class="abnf">
-            wkt-type             = (ALPHA / DIGIT) *(ALPHA / DIGIT / other)
-          </pre>
-          <p class=note>
-            The [[[NFC-RTD]]] defines every type in the [=well-known type records=]
-            and <a>external type records</a> in terms of URNs, but only a subset of
-            the URN is actually stored in the <a>NDEF record</a>'s <a>TYPE field</a>,
-            which corresponds to the above two ABNFs.
-          </p>
+          <a>external type record</a> that satisfies the
+          <a>validate external type</a> steps.
```

Comment:
"satisfies" doesn't really cut it, right?

Reply:
I don't think so.

Maybe you can write the ABNF in words instead. 

which is a string consisting of a domain name and a custom ASCII name, separated by a colon (':'). The string may not contain any whitespace character.

Reply:
Thanks, great!